ReClass.NET/Forms/MainForm.cs

```diff
@@ -617,6 +619,12 @@ private void searchForEqualValuesToolStripMenuItem_Click(object sender, EventArg
 				case UInt64Node node:
 					comparer = new LongMemoryComparer(ScanCompareType.Equal, (long)node.ReadValueFromMemory(selectedNode.Memory), 0);
 					break;
+				case UIntPtrNode node:
```

Comment:
That's not correct? If you have a x86 process it should be an `IntegerMemoryComparer` and not the 64bit `LongMemoryComparer`.